src/components/data-management/SpeciesSelector.jsx

```diff
@@ -1,20 +1,22 @@
-// aa
-
+import React from 'react';
+import PropTypes from 'prop-types';
 import {
   Select, Typography, Space, Divider, Skeleton,
 } from 'antd';
-import React from 'react';
```

Comment:
Can you apply this more consistent import ordering to `ProjectDetails.jsx` also?

Reply:
done, reordered